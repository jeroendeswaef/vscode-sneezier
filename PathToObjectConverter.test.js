const PathToObjectConverter = require('./PathToObjectConverter');

describe('Testing conversion of paths to object', () => {
	test('Testing paths with lines', () => {
        expect(new PathToObjectConverter([{
            type: 'helperPoint',
            x: 60,
            y: 0
        }, {
            type: 'lineTo',
            x: 120,
            y: 0
        }, {
            type: 'lineTo',
            x: 180,
            y: 60
        }]).getObjects()).toMatchSnapshot();
    })
})

