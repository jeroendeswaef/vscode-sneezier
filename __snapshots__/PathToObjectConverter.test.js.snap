// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing conversion of paths to object Testing paths with lines 1`] = `
Array [
  Object {
    "type": "line",
    "x1": 60,
    "x2": 120,
    "y1": 0,
    "y2": 0,
  },
  Object {
    "type": "line",
    "x1": 120,
    "x2": 180,
    "y1": 0,
    "y2": 60,
  },
]
`;
